appserver/web/war-util/src/main/java/org/glassfish/web/loader/WebappClassLoader.java

```diff
@@ -267,7 +269,12 @@ public final class WebappClassLoader extends GlassfishUrlClassLoader
      * but no defined repositories.
      */
     public WebappClassLoader(ClassLoader parent) {
+        this(parent, "unknown");
```

Comment:
I am against this  change, it is confusing and doesn't bring a value.

Reply:
You mean it doesn't bring any value that the log message says which application is related to the web app classloader? I thought it's very important, otherwise the new INFO message just says that some web app classloader was closed.

Reply:
Alternatively, I can just change the log level of the original message rom INFO to DEBUG. I don't think it's important to log an INFO message when classloader is closed, what do you think?

Reply:
I mean that it logs the value just in one case of five possible creations of the classloader. If it should know the name of the application, it should know it always, without any "unknown" value.